stock_landed_cost_valuation_real_time_bypass_check/models/stock_landed_cost.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, models
+from odoo.exceptions import UserError
+
+
+class LandedCost(models.Model):
+    _inherit = "stock.landed.cost"
+
+    def compute_landed_cost(self):
+        _self = self.with_context(bypass_valuation_real_time=True)
+        return super(LandedCost, _self).compute_landed_cost()
+
+    def get_valuation_lines(self):
+        if not self.env.context.get("bypass_valuation_real_time"):
+            return super().get_valuation_lines()
+        lines = []
+
+        for move in self.mapped("picking_ids").mapped("move_lines"):
```

Comment:
Why do you need to do all this stuff?

Reply:
According to https://github.com/odoo/odoo/blob/6f7e9a93048155e403e6cc56d24595c281297124/addons/stock_landed_costs/models/stock_landed_cost.py#L215-L216
we need to avoid that lines are not generated for products with category `valuation='real_time'`

Can you think of another possibility to do this better?

Reply:
I have tried to avoid completely redoing this function and changing product categories to `real_time` previously, but that causes a big change that is NOT what we want (https://github.com/odoo/odoo/blob/13.0/addons/stock_account/models/product.py#L728-L770). I think it is better to redo `get_valuation_lines()` function than to avoid by context those changes in the category of a product.
What do you think?